ui/app/models/pki/urls.js

```diff
@@ -0,0 +1,42 @@
+import Model, { attr } from '@ember-data/model';
+import { withFormFields } from 'vault/decorators/model-form-fields';
+import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
+
+@withFormFields()
+export default class PkiUrlsModel extends Model {
+  // This model uses the backend value as the model ID
+  get useOpenAPI() {
+    return true;
+  }
+  getHelpUrl(backendPath) {
+    return `/v1/${backendPath}/config/urls?help=1`;
+  }
```

Comment:
I'm wondering if using openAPI is necessary here since you manually defined the attrs needed?

Reply:
Yes, Alex specifically asked that we use openAPI for this section since they will likely be shipping new options soon. I only overrode 3 of the 4 options currently present on the form